### DISCUSSION TITLE
API Sandbox for Sunbird-RC

### DISCUSSION THREAD

**Discussion by vrayulu** [Sunbird-RC/community#265] [General]
Hi @parthlawate ,

We need a Sunbird RC API sandbox to use and understand the APIs. We intent to use this in Smart India Hackathon happening on 25-26 Aug 2022. Could you please provide us this sandbox with sample schemas loaded and the steps to procure an API key to use the APIs.

Regards,
Rayulu

**Comment by parthlawate** [comment]
Hello @vrayulu 

We currently have demos setup & related documentation that you can check here https://docs.sunbirdrc.dev/  (check the reference solutions & demos section)

For the students to setup their own instance please find the docs here https://docs.sunbirdrc.dev/developer-documentation/installation-guide 

We don't have a Sandbox as such right now. We'll check & revert how we can set one up quickly.  

cc @dileepbapat @tejash-jl

**Reply by pramodkvarma** [reply]
This is a great opportunity for RC community to get students to build innovations.

**Comment by parthlawate** [comment]
@vrayulu We are trying to see if we can quickly Spin up a self service sandbox. We'll extend this further in later phases as a full UI to deploy full registry backend, frontend including configuration..  

However as a backup plan, is it possible to get details of the teams who need this via say a Google Form ?

**Reply by vrayulu** [reply]
@parthlawate - we don't have the information about the teams as of now. We will meet them on 25th morning and identify who can potentially use SB-RC in their respective problems.

**Reply by vrayulu** [reply]
we are thinking to take a few sets of API keys & user credentials with us to the hackathon and provide it to the participants to use SB-RC.

**Comment by tejash-jl** [comment]
@vrayulu I have created a sandbox environment for SB-RC. I have also shared the API keys/secrets with you on discord. Please check and let me know.

**Comment by parthlawate** [comment]
@vrayulu can you mark this solved please ?

**Reply by vrayulu** [reply]
Thanks @tejash-jl and @parthlawate for sharing the details. We are able to use the sandbox and this thread can be closed now.

@parthlawate - Is there a way to close/lock the discussion thread?

**Reply by parthlawate** [reply]
Guess not on this type of post !. So we'll consider it closed. 

For some types Github allows mark as closed
